### ISSUE TITLE
Filter experiments by float param

### ISSUE THREAD

**Issue by mateuszpieniak** [polyaxon/polyaxon#670]
### Describe the bug
Although I have experiments listed in the group with some learning rate as depicted below,
![image](https://user-images.githubusercontent.com/31375424/73248555-75d3fd00-41b3-11ea-8fac-3ae9bdc3942a.png)

filtering by some learning rate value doesn't work.

![image](https://user-images.githubusercontent.com/31375424/73248689-b2075d80-41b3-11ea-9c06-e38c42e4bedf.png)

### To Reproduce
Run a grid of experiments with different learning rates. Apply the filter from the screenshot.

### Expected behavior
Filter works.

### Environment
* Polyaxon 0.5.6

**Comment by mouradmourafiq** [comment]
Can you upgrade to v0.6.0, and see if it works?

**Comment by mateuszpieniak** [comment]
@mouradmourafiq Sorry, but currently I don't have such a possibility.

**Comment by mouradmourafiq** [comment]
I will keep this as a bug for the moment, the reason I asked is because there was a fix for the sort logic.

**Comment by mateuszpieniak** [comment]
@mouradmourafiq It doesn't work in v0.6.0.

**Comment by mouradmourafiq** [comment]
Ok this needs a bit more investigation, it could be related to how the Postgres is storing floating points?
One way to check is to look at the value using a sql query, will keep the bug around until I have a better solution.

**Comment by mouradmourafiq** [comment]
I think I know what could be the issue, it's probably a floating point related problem at the db level, I guess one way to find out is to do a comparison, i.e. `metrics.learning_rate:>0.000019, metrics.loss:<0.000021`

**Comment by mouradmourafiq** [comment]
One other thing to mention, is that in the previous version of the dashboard, the values were rounded by default, because the table had limited width, which is not the case in the new version:

![Screenshot 2020-05-02 at 15 24 42](https://user-images.githubusercontent.com/1261626/80865340-12151080-8c89-11ea-885f-8091f48d7375.png)

Not only it's possible to enlarge the columns, there's an quick expansion button.

And by using the full float value, the filter actually works:

![Screenshot 2020-05-02 at 15 26 09](https://user-images.githubusercontent.com/1261626/80865377-50123480-8c89-11ea-975b-abfffce0d603.png)

So, no need to try the comparison from the previous comment, the bug was coming from the rounding. Closing the issue as fixed.
